### PR TITLE
[IT-1967] Cleanup bucket access

### PR DESCRIPTION
**Pull request by zaro0508**
We are removing access to sandbox and scicomp AWS accounts which means that the ARN for those users will no longer be valid.  We are switching to use another AWS account (identity-central) to allow cross account access to buckets.  This change is to update user access to scicomp buckets.

* Remove users that are no longer Sage employees
* Update ARN to refrence users in identity-central account.

Now users must first login to org-sagebase-identitycentral AWS account to access scicomp buckets.

### CODE REVIEW

File: config/prod/JKG-SciComp-S3.yaml [hunk 1]
```diff
@@ -26,4 +26,3 @@ parameters:
   # (Optional) Allow accounts, groups, and users to access bucket (default is no access).
   GrantAccess:
     - 'arn:aws:iam::325565585839:root'   # Required ARN for a synapse bucket
-    - 'arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/jake.gockley@sagebase.org'
```
Comment (zaro0508) [inline config/prod/JKG-SciComp-S3.yaml:29]: Jake's gone.  Should this be owned by someone that's still around?

**Comment by allaway** [comment]
Changes to my bucket ([config/prod/nf-syn23664726-s3.yaml](https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/540/files#diff-4e1801cee3b3ebfa5e0b78552391a1d706e96ead9f8c4445df8b4ec0918c22da)) LGTM

**Review by allaway — APPROVED** [review]

**Review by danlu1 — APPROVED** [review]
The change made for [PsychENCODE-Globus-S3.yaml](https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/540/files#diff-31737ac46489e0f9198903d23bffa72b8faabc6a874f5635f81b7ab454b523f2) is correct.

**Review by adamjtaylor — APPROVED** [review]

**Review by philerooski — APPROVED** [review]

**Review by brucehoff — APPROVED** [review]

File: config/prod/JKG-SciComp-S3.yaml (hunk 1 above)
Comment (wpoehlm) [inline config/prod/JKG-SciComp-S3.yaml:29]: As per conversation with Khai and Anna on slack, this bucket can be deleted completely

**Review by wpoehlm — APPROVED** [review]

**Pull request merged**